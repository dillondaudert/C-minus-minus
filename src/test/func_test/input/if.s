	.section	.rodata
.int_wformat: .string "%d\n"
.str_wformat: .string "%s\n"
.int_rformat: .string "%d"
.string_const0: .string "enter a:"

	.globl main
	.type main @function

main:	nop
	pushq	%rbp
	movq	%rsp, %rbp
	subq	$4, %rsp
	movl	$.string_const0, %ebx
	movl	%ebx, %esi
	movl	$0, %eax
	movl	$.str_wformat, %edi
	call	printf
	leave
	ret
	.size	main, .-main

