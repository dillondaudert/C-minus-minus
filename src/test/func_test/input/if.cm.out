st_init created the symbol table hash map
Adding new symbol: main, main, offset: 0, type: 2, size: 4, arr: 0
st_add added main at position 0
Added process main to stable

Added a to $$names, count=1
Adding new symbol: a, _gp, offset: 0, type: 0, size: 0, arr: 0
st_add added a at position 64
updating symbol a
Symbol 64:a found
Symbol a updated with size 4, offset 0


Adding new symbol: .string_const0, .string_const0, offset: 0, type: 0, size: 9, arr: 0
st_add, added .string_const0 to collision LL at 0










Freed symbol at position 0
Freed LL symbol at position 0
Freed link in collision LL at 0
Freed symbol at position 64
Freed symbol table
